keyboards/v60_type_r/config.h

```diff
@@ -50,7 +50,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define BACKLIGHT_PIN F7
 //#define BACKLIGHT_BREATHING
 #define BACKLIGHT_LEVELS 3
-#define BACKLIGHT_ON_STATE 0
+#define BACKLIGHT_ON_STATE 1
```

Comment:
This should be left alone. 

If needed, it should be either undef'ed and redefined at the keymap level, or a separate PR should be submitted.